src/utils/urlParseHelper.ts

```diff
@@ -3,42 +3,47 @@ interface IScmRepositoryUrl {
 }
 
 export interface IParsedUrl {
-  url: string;
+  webUrl: string;
+  scmRepositoryUrl: string;
   name: string;
 }
 
 /**
  * Parses internal SCM Repository URL to Gerrit gitweb link of the SCM Repository.
  *
  * @param url - The internalUrl to be parsed
- * @returns Object containing parsed URL and display name representing URL
+ * @returns Object containing scmRepository URL, parsed URL and display name representing URL
  *  */
 export const parseInternalScmRepositoryUrl = ({ url }: IScmRepositoryUrl): IParsedUrl => {
   const protocol = url.split('://')[0];
   const base = url.split('://')[1].split('/')[0];
   const project = url.split(base + (['https', 'http'].includes(protocol) ? '/gerrit/' : '/'))[1];
-  return { url: 'https://' + base + '/gerrit/gitweb?p=' + project + ';a=summary', name: 'Gerrit' };
+  return { scmRepositoryUrl: url, webUrl: 'https://' + base + '/gerrit/gitweb?p=' + project + ';a=summary', name: 'Gerrit' };
 };
 
 /**
  * ParsesSCM Repository URL to gitweb link of the SCM Repository.
  *
  * @param url - The externalUrl to be parsed
- * @returns  Object containing parsed URL and display name representing URL
+ * @returns  1) null when url was not successfully parsed, otherwise
+ * 2) object containing parsed URL and display name representing URL
```

Comment:
`Object containing scmRepository URL, parsed URL and display name representing URL`